Firestore/Source/Core/FSTFirestoreClient.h

```diff
@@ -33,7 +37,6 @@
 @class FSTQuery;
```

Comment:
I think FSTDatabaseInfo and FSTDatabaseID were removed in master and no longer necessary.